plugins/pcgc_cax/pcgc_cax.go

```diff
@@ -28,9 +28,9 @@ const (
 var defaultConfig = types.PluginConfig{
 	Key:                "",
 	Scheme:             "https",
-	Endpoint:           "cax.devnet.clearmatics.network", // todo: replace it with cax.piccadilly.autonity.org for piccadilly network.
-	Timeout:            10,                               //10s
-	DataUpdateInterval: 3600,                             //3600s, todo: if there is a rate limit for the CAX service, we'd need to set it properly.
+	Endpoint:           "cax.devnet.clearmatics.network",
```

Comment:
```suggestion
	Endpoint:           "cax.piccadilly.autonity.org",
```
@Jason-Zhangxin-Chen edit to change to the root exchange URI for Piccadilly.

(Note the URI for calls to the SIM API  is: https://cax.piccadilly.autonity.org/api/

I assume you want the root URI not the API URI - unless you are calling an API function?

FYSA @aiman

Reply:
> @Jason-Zhangxin-Chen edit to change to the root exchange URI for Piccadilly.
> 
> (Note the URI for calls to the SIM API is: https://cax.piccadilly.autonity.org/api/
> 
> I assume you want the root URI not the API URI - unless you are calling an API function?
> 
> FYSA @aiman

I got some timeout error from switching to this host `cax.piccadilly.autonity.org` for the data from CAX service of piccadilly, please confirm that service is deployed, but anyway it is configurable. Please confirm such change, and I will do some testing on the new service endpoint before we go ahead.

Reply:
The CAX for Piccadilly is not deployed yet, the deployment is in-progress.

Reply:
@cmjc @raj-shekhar1, to support CAX service of both dev-net and piccadilly, I introduced a build flag in bafb1057317c399f97e4bff3736cc88d2cb9d3af, @raj-shekhar1 you can build oracle server and the plugins for dev-net by `make autoracle-dev` while `make autoracle` will build things for piccadilly. This will make things be simple now.